src/services/proposals.ts

```diff
@@ -30,3 +30,70 @@ export async function createProposalOnDb(mySql: ServerlessMysql, data: CreatePro
 
   return { proposalId };
 }
+
+export interface IProposalSqlRow {
+  proposal: string,
+  hashed_auth_password: string,
+  auth_password_salt: string,
+  version: number,
+  partial_tx: string,
+  signatures: string,
+  history: string,
+  created_at: string,
+  updated_at: string,
+}
+
+export interface IGetProposalFromDb {
+  id: string,
+  hashedAuthPassword: string,
+  authPasswordSalt: string,
+  partialTx: string,
+  signatures: string,
+  timestamp: string,
+  version: number,
+  history: {partialTx: string, timestamp: string}[]
+}
+
+export async function getProposalFromDb(mySql: ServerlessMysql, proposalId: string) {
```

Comment:
This query should return exactly 1 row, correct?
Should we have an error if we return more than one row?
This would protect from future changes on the query or possible database errors, i.e. `proposalId` is not the primary key.